.github/workflows/plugin-updater.yml

```diff
@@ -26,8 +22,8 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
           REPOSITORY: "Flow-Launcher/Flow.Launcher.PluginsManifest"
-          PLUGINS_JSON: "plugins.json"
-          PLUGINS_MARKDOWN: "plugins.md"
+          PLUGIN_JSON: "plugins.json"
+          PLUGIN_MARKDOWN: "plugins.md"
```

Comment:
this plugins md should be plugins.md? if it is referencing the md file in the docs repo

Reply:
fine, another error.

Reply:
I use variable `PLUGIN_JSON` and `PLUGIN_MARKDOWN` in script no `s`